runtime/render_a/src/sys/d3d/d3d_texture.cpp

```diff
@@ -324,15 +324,15 @@ bool CTextureManager::ConvertTexDataToDD(uint8* pSrcData, PFormat* SrcFormat, ui
 #else
  	D3DFORMAT D3DSrcFormat = d3d_PFormatToD3DFormat(SrcFormat); assert(D3DSrcFormat != D3DFMT_UNKNOWN);
 	D3DFORMAT D3DDstFormat = QueryDDFormat1(eDstType, nDstFlags); assert(D3DDstFormat != D3DFMT_UNKNOWN);
-	
+
 	// Create a quick little surface to convert into....
-	LPDIRECT3DSURFACE9 pD3DDstSurface = NULL; LPDIRECT3DTEXTURE9 pD3DDstTexture = NULL;
-	HRESULT hResult = PD3DDEVICE->CreateTexture(DstWidth,DstHeight,1,NULL,D3DDstFormat,D3DPOOL_SYSTEMMEM,&pD3DDstTexture);
-	if (hResult != D3D_OK || !pD3DDstTexture) 
+	LPDIRECT3DSURFACE9 pD3DDstSurface = nullptr; LPDIRECT3DTEXTURE9 pD3DDstTexture = nullptr;
+	HRESULT hResult = PD3DDEVICE->CreateTexture(DstWidth,DstHeight,1,nullptr,D3DDstFormat,D3DPOOL_SYSTEMMEM,&pD3DDstTexture);
```

Comment:
This is causing an error in Visual Studio, as the fourth argument is expecting a DWORD. I recommend just setting this to 0.

Reply:
yeah my mistake, this code isn't compiled on Linux, so I missed it.